### ISSUE TITLE
[PowerPC][FreeBSD][LLDB] /wrkdirs/usr/ports/devel/llvm-devel/work-default/llvm-project-596c5581556cbd4103d982066fb76e745fb8480e/lldb/source/Plugins/Process/FreeBSD/NativeRegisterContextFreeBSD_powerpc.cpp:184:28: error: no member named 'getValue' in 'std::optional<lldb_private::process_freebsd::NativeRegisterContextFreeBSD_powerpc::RegSetKind>'

### ISSUE THREAD

**Issue by pkubaj** [llvm/llvm-project#60401]
LLDB in the current main branch fails to build with:
```console
$ /usr/bin/c++ -DHAVE_ROUND -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/wrkdirs/usr/ports/devel/llvm-devel/work-default/.build/tools/lldb/source/Plugins/Process/FreeBSD -I/wrkdirs/usr/ports/devel/llvm-devel/work-default/llvm-project-596c5581556cbd4103d982066fb76e745fb8480e/lldb/source/Plugins/Process/FreeBSD -I/wrkdirs/usr/ports/devel/llvm-devel/work-default/llvm-project-596c5581556cbd4103d982066fb76e745fb8480e/lldb/include -I/wrkdirs/usr/ports/devel/llvm-devel/work-default/.build/tools/lldb/include -I/wrkdirs/usr/ports/devel/llvm-devel/work-default/.build/include -I/wrkdirs/usr/ports/devel/llvm-devel/work-default/llvm-project-596c5581556cbd4103d982066fb76e745fb8480e/llvm/include -I/usr/local/include/python3.9 -I/wrkdirs/usr/ports/devel/llvm-devel/work-default/llvm-project-596c5581556cbd4103d982066fb76e745fb8480e/llvm/../clang/include -I/wrkdirs/usr/ports/devel/llvm-devel/work-default/.build/tools/lldb/../clang/include -I/wrkdirs/usr/ports/devel/llvm-devel/work-default/llvm-project-596c5581556cbd4103d982066fb76e745fb8480e/lldb/source -I/wrkdirs/usr/ports/devel/llvm-devel/work-default/.build/tools/lldb/source -isystem /usr/local/include/libxml2 -O2 -pipe -DNDEBUG -fstack-protector-strong -isystem /usr/local/include -fno-strict-aliasing  -DNDEBUG -isystem /usr/local/include -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wno-noexcept-type -Wnon-virtual-dtor -Wdelete-non-virtual-dtor -Wsuggest-override -Wstring-conversion -Wmisleading-indentation -Wctad-maybe-unsupported -fdiagnostics-color -ffunction-sections -fdata-sections -Wno-deprecated-declarations -Wno-unknown-pragmas -Wno-strict-aliasing -Wno-deprecated-register -Wno-vla-extension -O2 -pipe -DNDEBUG -fstack-protector-strong -isystem /usr/local/include -fno-strict-aliasing  -DNDEBUG -isystem /usr/local/include  -DNDEBUG  -fno-exceptions -std=c++17 -MD -MT tools/lldb/source/Plugins/Process/FreeBSD/CMakeFiles/lldbPluginProcessFreeBSD.dir/NativeRegisterContextFreeBSD_powerpc.cpp.o -MF tools/lldb/source/Plugins/Process/FreeBSD/CMakeFiles/lldbPluginProcessFreeBSD.dir/NativeRegisterContextFreeBSD_powerpc.cpp.o.d -o tools/lldb/source/Plugins/Process/FreeBSD/CMakeFiles/lldbPluginProcessFreeBSD.dir/NativeRegisterContextFreeBSD_powerpc.cpp.o -c /wrkdirs/usr/ports/devel/llvm-devel/work-default/llvm-project-596c5581556cbd4103d982066fb76e745fb8480e/lldb/source/Plugins/Process/FreeBSD/NativeRegisterContextFreeBSD_powerpc.cpp
/wrkdirs/usr/ports/devel/llvm-devel/work-default/llvm-project-596c5581556cbd4103d982066fb76e745fb8480e/lldb/source/Plugins/Process/FreeBSD/NativeRegisterContextFreeBSD_powerpc.cpp:184:28: error: no member named 'getValue' in 'std::optional<lldb_private::process_freebsd::NativeRegisterContextFreeBSD_powerpc::RegSetKind>'
  RegSetKind set = opt_set.getValue();
                   ~~~~~~~ ^
/wrkdirs/usr/ports/devel/llvm-devel/work-default/llvm-project-596c5581556cbd4103d982066fb76e745fb8480e/lldb/source/Plugins/Process/FreeBSD/NativeRegisterContextFreeBSD_powerpc.cpp:218:28: error: no member named 'getValue' in 'std::optional<lldb_private::process_freebsd::NativeRegisterContextFreeBSD_powerpc::RegSetKind>'
  RegSetKind set = opt_set.getValue();
                   ~~~~~~~ ^
2 errors generated.
```

**Comment by MaskRay** [comment]
Hopefully fixed by 003f2b379d52832171eaa6272e3a2b1cabcd6f4e. Please reopen if the issue is still present...

**Comment by pkubaj** [comment]
Can you merge that to 16.0.0?
